### DISCUSSION TITLE
「开源之史」系列之一/七/四：软件版权的自我保护完善还是霸权的扩张 —— 二进制被正式纳入版权法 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#52] [Announcements]
# 「开源之史」系列之一/七/四：软件版权的自我保护完善还是霸权的扩张 —— 二进制被正式纳入版权法 - 「开源之道」

https://opensourceway.community/posts/history-of-open-source/02-04-001-copyright-cover-binary/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

交通广播连载小话本体裁...

每一个决定瞬间, 都是有其历史上的巧合/被迫/应激/...的积累,
只是, 这种事件的阐述, 值得以新闻报道的格式来尝试卡片式记录,

也就是说 5W1H: 

+ When 何时
+ Where 何地
+ What 何事
+ Why 何因
+ Who 何人
+ How 如何

都得完备, 不好说一半就停下来了, 完全可以链接到其它相关文章/故事中哪,
将 blog 当成 wiki/双链笔记/图书/.. 来写是没问题的...

所以, RMS 干了什么? 这就值得有个链接了...


SEE: [5W1H \- Woodpecker Wiki for CPUG](https://wiki.woodpecker.org.cn/moin/5W1H)
